containers/release-candidate/src/daffodil-release-candidate

```diff
@@ -300,6 +301,23 @@ case $PROJECT_REPO in
     yarn package
     cp *.vsix $DAFFODIL_RELEASE_DIR/bin/
 
+    echo "Building Extension Docs..."
+    git clone https://github.com/apache/daffodil-vscode.wiki.git .wiki
+    for markdown_file in $(echo "\
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code \
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code:-Development \
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code:-Roadmap \
+      Home"
+    ); do
+      pandoc -t html -f markdown -o "${markdown_file}.html" ".wiki/${markdown_file}.md"
```

Comment:
I thought the original goal was to have a single PDF easily downloadable for offline documentation? But this just creates multiple html pages, which I think are really just snapshot of the wiki. And the fomatting isn't as nice. For example, the github pages have a navigation menu on he right of each page, and the images are scaled to fit.

So I guess my question is how important is this snapshotting of wiki documentation? Seems to me most of the time people will always be using the latest version of the extension and go to the github wiki. It feels different from the Daffodil JAPI/SAPI where often times upgrading to a new version really does incur a cost so people do need API docs of older versions. And Java/Scala docs also aren't easily available since they need to be generated, unlike the github wiki.

Reply:
Oh if that is what we want it makes sense to me. I am not sure if I can combine into one single page. From the code I made it from it seemed like it was doing page by page so that is what I did. But I can try testing what you mention here. @scholarsmate Are you able to better explain what was desired for saving off the documentation from the wiki?

Reply:
Yeah I don't know though about a single PDF is that possible with pandoc?

Reply:
If snapshotting the wiki still serves a useful purpose, there are other tools you can look at such as [github-wikito-converter](https://github.com/yakivmospan/github-wikito-converter) which can convert the entire wiki to a single HTML or PDF file in one go:

```shell
sudo npm install -g github-wikito-converter
git clone https://github.com/apache/daffodil-vscode.wiki.git .wiki
gwtc --file-name <filename> --format <html|pdf> --title "<wiki title>" .wiki
```

Reply:
So I had some issues with getting pdf to work completely with that tool. However, I was able to produce a single docx file for all Wiki markdown files.